pkg/operator/controller/status.go

```diff
@@ -148,6 +151,11 @@ func computeOperatorStatusConditions(conditions []configv1.ClusterOperatorStatus
 			"%d ingress controllers available, want %d",
 			ingressesAvailable, numIngresses)
 	}
+	// LastTransitionTime is updated when running and configured
+	// versions are not equal to indicate an upgrade has occurred.
+	if !versionsEqual {
+		progressingCondition.LastTransitionTime = metav1.Now()
```

Comment:
Would it make any sense to set a reason (e.g. VersionMismatch) and a message (e.g. "From version X to version Y")?

Reply:
Yeah, better to set a message than to fiddle with last transition time.

Reply:
CIO needs to reset `LastTransitionTime` to fix [Bug 1695210](https://bugzilla.redhat.com/show_bug.cgi?id=1695210). After talking with @pravisankar yesterday, we thought this requirement should not trigger a change to the Operator's `Progressing` condition since ingress controllers are not being affected by the Operator upgrade. According to the bug, changing the `Progressing` condition is not required, but updating `LastTransitionTime` is. Thoughts?

Reply:
Updating `Message` using `setOperatorStatusCondition` will update `LastTransitionTime` and should cause `operatorStatusesEqual` to consider the status unequal so `syncOperatorStatus` will update it.

Reply:
Ahh... I did not know that. Thanks for the heads-up. I'll update and resubmit the PR.